test/lshr-constexpr.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: llvm-as %s -o %t.bc
 ; RUN: llvm-spirv %t.bc -o %t.spv
-; RUN: llvm-spirv -r %t.spv -o %from-spirv.bc
-; RUN: llvm-dis %from-spirv.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
+; RUN: llvm-spirv -r %t.spv -o %t.from-spirv.bc
```

Comment:
Nitpick: In other tests it is typically called `t.rev.bc`

Reply:
Thanks, good point!  Fixed.